operator/v1/types_network.go

```diff
@@ -79,9 +79,11 @@ type NetworkSpec struct {
 	// +listMapKey=name
 	AdditionalNetworks []AdditionalNetworkDefinition `json:"additionalNetworks,omitempty"`
 
-	// disableMultiNetwork specifies whether or not multiple pod network
-	// support should be disabled. If unset, this property defaults to
-	// 'false' and multiple network support is enabled.
+	// disableMultiNetwork enables multiple network support when the OVN-Kubernetes Container Network Interface (CNI)
+	// plugin sets the parameter to the default value of 'false'.During cluster installation only, you can use a 
+	// third-party CNI, such as Multus CNI, to set the disableMultiNetwork parameter to 'true', and this setting disables
+	// multiple pod network support.Red{nbsp}Hat does not support third-party CNIs.
```

Comment:
These docs are generated and used in many different places, ascii doc markup is not suitable here
```suggestion
	// multiple pod network support. Red Hat does not support third-party CNIs.
```

Reply:
:heavy_check_mark: